contracts/core/Core.sol

```diff
@@ -16,13 +16,14 @@
 
 pragma solidity 0.4.24;
 
-
-import { Ownable } from "zeppelin-solidity/contracts/ownership/Ownable.sol";
-import { SafeMath } from "zeppelin-solidity/contracts/math/SafeMath.sol";
+import { CoreInternal } from "./CoreInternal.sol";
+import { CoreState } from "./lib/CoreState.sol";
 import { ISetFactory } from "./interfaces/ISetFactory.sol";
 import { ISetToken } from "./interfaces/ISetToken.sol";
 import { ITransferProxy } from "./interfaces/ITransferProxy.sol";
 import { IVault } from "./interfaces/IVault.sol";
+import { Ownable } from "zeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
nit: the package paths (i.e. non-relative paths) should remain at top of file as a general style guide

Reply:
+1